src/cascadia/Remoting/SummonWindowBehavior.h

```diff
@@ -0,0 +1,35 @@
+/*++
+Copyright (c) Microsoft Corporation
+Licensed under the MIT license.
+
+Class Name:
+- SummonWindowBehavior.h
+
+Abstract:
+- TODO!
+
+--*/
+
+#pragma once
+
+#include "SummonWindowBehavior.g.h"
+#include "../cascadia/inc/cppwinrt_utils.h"
+
+namespace winrt::Microsoft::Terminal::Remoting::implementation
+{
+    struct SummonWindowBehavior : public SummonWindowBehaviorT<SummonWindowBehavior>
+    {
+    public:
+        SummonWindowBehavior() = default;
+        WINRT_PROPERTY(bool, MoveToCurrentDesktop, true);
+
+    public:
+        SummonWindowBehavior(const Remoting::SummonWindowBehavior& other) :
+            _MoveToCurrentDesktop{ other.MoveToCurrentDesktop() } {};

```

Comment:
nit: you can move this above the WINRT_PROPERTY to not have to say `public` again. Probably makes sense to be next to the other ctor too.

Reply:
I'm not sure that I can actually - technically, `_MoveToCurrentDesktop` hasn't been declared above the WINRT_PROPERTY, so trying to use it there will confuse the compiler